.changes/v2.16.0/483-bug-fixes.md

```diff
@@ -0,0 +1 @@
+* XML tags fixed for correct unmarshalling of resourcepoolrefs in adminvdc
```

Comment:
nit:
```suggestion
* XML tags fixed for correct unmarshalling of `ResourcePoolRefs` in `AdminVdc` [GH-483]
```

Reply:
Please also use the correct capitalization in the PR name as well.